code/modules/shuttle/shuttle_manipulator.dm

```diff
@@ -160,6 +163,10 @@
 					break
 
 		if("preview")
+			if(cooldown_do_not_edit + PREVIEW_OR_SHUTTLE_SPAWN_COOLDOWN > world.time) 
+				atom_say("Please wait while the shuttle manipulator recalibrates...")
```

Comment:
make this a to_chat to the admin pressing buttons. Players intentionally can't see what admins are doing in the menu.

Reply:
Done